### DISCUSSION TITLE
Property types

### DISCUSSION THREAD

**Discussion by bwalsh** [ncihtan/htan-portal#313] [Q&A]
In the jsonld document, most properties are defined as   `"@type": "rdfs:Class"`  ( vs. `"@type": "rdf:Property"`)

I noticed this defining numerics in the code vs in the schema

https://github.com/ncihtan/htan-portal/blob/f496b6173fc7e2553c02e79fe239fe879e0ac56f/lib/dataSchemaHelpers.ts#L139-L167

I also noticed type information embedded in comments. Parsing comments, I noticed the following "type" information in comments:

```
'"uint16"', "'null'", '(True/False)', '(mL).', '(mm).', '(um)', '(um).', ')', 'Integer', 'Integer.', 'String', 'Text.',   'URL',   '[Number]', '[URL]', '[number]', 'age', 'birthday.', 'centimeters.', 'cm', 'comma-separated', 'day.', 'days', 'days.', 'file', 'file.', 'filenames', 'files', 'grams.', 'kilograms.', 'length', 'list', 'mm', 'null', 'number', 'time', 'timepoint.'
```
Is there a convention on primitive type location?

**Comment by alliecreason** [comment]
Bump @milen-sage, we would appreciate your feedback here.
